shotover-proxy/src/transforms/cassandra/sink_cluster.rs

```diff
@@ -32,29 +33,44 @@ impl CassandraSinkClusterConfig {
         Ok(Transforms::CassandraSinkCluster(CassandraSinkCluster::new(
             self.first_contact_points.clone(),
             chain_name,
+            self.data_center.clone(),
             tls,
         )))
     }
 }
 
 pub struct CassandraSinkCluster {
     contact_points: Vec<String>,
-    outbound: Option<CassandraConnection>,
+    contact_point_handshake: Vec<Message>,
+    contact_point_handshake_complete: bool,
+    contact_point_connection: Option<CassandraConnection>,
     chain_name: String,
     failed_requests: Counter,
     tls: Option<TlsConnector>,
     pushed_messages_tx: Option<mpsc::UnboundedSender<Messages>>,
+    peer_table: FQName,
+    data_center: String,
+    nodes: Vec<CassandraNode>,
```

Comment:
I think we should put this stuff (cluster metadata) in a struct.

Reply:
which stuff is that?
peer_table and data_center are immutable while nodes is mutable so I dont think they belong together?

Reply:
I anticipate us moving stuff out of this file into a struct holding the cluster stuff and doing things like handling different routing strategies and topology rebuilding in there rather than the main transform file.

But probably premature optimisation so lets go ahead with this for now.